routes/html-routes.js

```diff
@@ -1,14 +1,30 @@
-const path = require('path');
 const router = require('express').Router();
+const exphbs = require("express-handlebars");
 
 router.get('/', (req, res) => {
-  res.sendFile(path.join(__dirname, '../public/index.html'));
+  res.render("index");
+});
+
+
+router.get('/:token', (req, res) => {
```

Comment:
is this for testing, you used token?

Reply:
this is for real. this needs to look up the token from the database and return the user data.